mgmt_cli_test.py

```diff
@@ -282,7 +282,7 @@ def test_client_encryption(self):
         manager_tool = mgmt.get_scylla_manager_tool(manager_node=self.monitors.nodes[0])
         mgr_cluster = manager_tool.add_cluster(name=self.CLUSTER_NAME+"_encryption", db_cluster=self.db_cluster,
                                                auth_token=self.monitors.mgmt_auth_token)
-        self._generate_load()
```

Comment:
same here... why it's repair won't work while stress is running ?

Reply:
same as above, in addition that here, the c-s load fails when we change the cluster to work with client encryption.